test-utils/src/lib.rs

```diff
@@ -66,6 +66,25 @@ pub type QuadFelt = vm_core::QuadExtension<Felt>;
 // CONSTANTS
 // ================================================================================================
 
+/// The event ID for the event where we push the falcon signature on the advice stack.
+///
+/// Pushes values onto the advice stack which are required for verification of a DSA in Miden
+/// VM.
+///
+/// Inputs:
+///   Operand stack: [PK, MSG, ...]
+///   Advice stack: [...]
+///
+/// Outputs:
+///   Operand stack: [PK, MSG, ...]
+///   Advice stack: \[DATA\]
+///
+/// Where:
+/// - PK is the digest of an expanded public.
+/// - MSG is the digest of the message to be signed.
+/// - DATA is the needed data for signature verification in the VM.
+///
+/// The advice provider is expected to contain the private key associated to the public key PK.
```

Comment:
Seems like this was copied here by mistake?

Reply:
Oops yes, I moved the constant but forgot the docstring. Removed